examples/basic-sc/README.md

```diff
@@ -50,107 +50,67 @@ objects would be set by the different personas in the system:
-
-# THIS CLUSTERROLEBINDING IS FOR DEMO PURPOSES ONLY - THIS WILL GRANT MORE PERMISSIONS THAN NECESSARY
-#
-kubectl create clusterrolebinding gitops-toolkit-admin \
-  --clusterrole=cluster-admin \
-  --serviceaccount=gitops-toolkit:default
-
-SOURCE_CONTROLLER_VERSION=0.17.0
-
-kapp deploy --yes -a gitops-toolkit \
-  --into-ns gitops-toolkit \
-  -f https://github.com/fluxcd/source-controller/releases/download/v$SOURCE_CONTROLLER_VERSION/source-controller.crds.yaml \
-  -f https://github.com/fluxcd/source-controller/releases/download/v$SOURCE_CONTROLLER_VERSION/source-controller.deployment.yaml
-```
-
-- [kapp-controller], for providing us with the ability of deploying multiple
+- [kapp-controller](https://carvel.dev/kapp-controller/docs/latest/install/),
+  for providing us with the ability of deploying multiple
   Kubernetes objects as a single unit
 
-```bash
-KAPP_CONTROLLER_VERSION=0.30.0
+- [knative-serving](https://knative.dev/docs/install/serving/install-serving-with-yaml/),
+  for being the runtime of the application we want to deploy.
 
-kapp deploy --yes -a kapp-controller \
-	-f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v$KAPP_CONTROLLER_VERSION/release.yml
-```
+4. A container image registry in which you have authorization to push images. The
+  credentials will be passed to kpack in the configuration steps below.
 
-- [knative-serving], for being the runtime of the application we want to deploy.
+5. [Tree](https://github.com/ahmetb/kubectl-tree), a tool that we will use to observe the objects created.
 
-```bash
-KNATIVE_SERVING_VERSION=0.26.0
+## Running the example in this directory
 
-kapp deploy --yes -a knative-serving \
-  -f https://github.com/knative/serving/releases/download/v$KNATIVE_SERVING_VERSION/serving-crds.yaml \
-  -f https://github.com/knative/serving/releases/download/v$KNATIVE_SERVING_VERSION/serving-core.yaml
-```
+This example uses two directories with sub-directories of Kubernetes resources:
+[../shared](../shared) and [this directory (.)](.).
 
-4. Authorization to push images to a container image registry
+The shared directory has a subdirectory for cluster-wide configuration: [../shared/cluster](../shared/cluster)
 
-As mentioned before, in this example we make use of `kpack` to build container
-images out of the source code and push to a container image registry,
-so we must provide a Kubernetes secret that contains the credentials for pushing to a
-registry (see [kpack secrets] to know more about how kpack makes use of it).
+There is a subdirectory of cartographer-specific files that an App Operator would submit
+in both this and in the shared directory:
+[./app-operator](./app-operator) and [../shared/app-operator](../shared/app-operator)
 
+Finally, in this and in the shared directory there is a subdirectory containing
+Kubernetes objects that a developer would submit: [./developer](./developer) and
+[../shared/developer](../shared/developer)
```

Comment:
It's a great idea to organize the content into folders and subfolders but this section feels confusing; what's the purpose of the `shared` folder? (I could infer it but would be useful to make it more explicit). If it's content that is shared among all the examples, we could avoid `and in the shared directory` so it could be easier to read

Reply:
You are correct, it is content that's shared among the examples. It's specified here because users will need to submit those objects. If we remove the mention of them, I'm concerned that users will skip that step and the example will not work.

Reply:
If you have ideas on how to clarify that language, I'm happy to incorporate them! Should we just use the same language as the paragraph above:

```
There is a subdirectory of cartographer-specific files that an App Developer would submit
in both this and in the shared directory:
```